### DISCUSSION TITLE
Webhook json data format?

### DISCUSSION THREAD

**Discussion by Pipen** [supabase/supabase#17879] [Questions]
I try to run a web hook whenever a new row is inserted in my table. 

My webhook reach out to slack to post a message about the new insert.
The problem is that I get a 400 with the message "no_text" in my _http_response table.

Curling works like a charm
`curl -X POST -H 'Content-type: application/json' --data '{"text":"Hello, World!"}' https://hooks.slack.com/services/T...../q.....`

So I assume that the data is sent in a different format than above. Anyone had success with this? Anything I can do about it?

Thanks in advance.

**Comment by GaryAustin1** [comment]
Here is the format.  It is fixed.
https://supabase.com/docs/guides/database/webhooks#payload
If you don't like the format you can always do your own trigger function and use pg_net or http extensions to send an http request.

**Comment by isabellateng** [comment]
how did you fix this? @Pipen having the same issue

**Comment by Pipen** [comment]
I created a very simple edge-function to post to slack.
